cloud/pkg/controller/controller/upstream.go

```diff
@@ -331,6 +331,24 @@ func (uc *UpstreamController) updateNodeStatus(stop chan struct{}) {
 					log.LOGGER.Warnf("message: %s process failure, update node failed with error: %s, namespace: %s, name: %s", msg.GetID(), err, getNode.Namespace, getNode.Name)
 					continue
 				}
+
+				resMsg := model.NewMessage(msg.GetID())
+				resMsg.Content = "update node status successfully"
+				nodeID, err := messagelayer.GetNodeID(msg)
+				if err != nil {
+					log.LOGGER.Warnf("message: %s process failure, get node id failed with error: %s", msg.GetID(), err)
+				}
+				resource, err := messagelayer.BuildResource(nodeID, namespace, model.ResourceTypeNode, name)
+				if err != nil {
+					log.LOGGER.Warnf("message: %s process failure, build message resource failed with error: %s", msg.GetID(), err)
+				}
+				resMsg.BuildRouter(constants.EdgeControllerModuleName, constants.GroupResource, resource, model.ResponseOperation)
+				err = uc.messageLayer.Response(*resMsg)
```

Comment:
condense it to:

if err := uc.messageLayer.Response(*resMsg); err != nil {
  ...
}

to save one line :)